docs/public/installation/zonemaster-backend.md

```diff
@@ -711,7 +718,69 @@ Now go back to "[Database configuration](#53-database-configuration-freebsd)"
 to create the database tables and then continue with the steps after that.
 
 
-## 9. Cleaning up the database
+## 9. Installation with Clickhouse (experimental)
+
+First follow the installation instructions for the OS in question, and then go
+to this section to install Clickhouse.
+
+
+### 9.1. Clickhouse (Rocky Linux)
+
+There is no specific installation instructions for Rocky Linux yet. In most
+parts the instructions for Debian/Ubuntu can be followed.
+
+
+### 9.2. Clickhouse (Debian/Ubuntu)
+
+> This is an experimental feature.
+
+To install Clickhouse, follow the instructions from the [official
+documentation](https://clickhouse.com/docs/en/install#available-installation-options).
+
+Configure Zonemaster::Backend to use the correct database engine:
+
+```sh
+sudo sed -i '/\bengine\b/ s/=.*/= Clickhouse/' /etc/zonemaster/backend_config.ini
+```
+
+> **Note:** See the [backend configuration] documentation for details.
+
+Install the Perl bindings (Clickhouse relies on the MySQL DBI).
+
+```sh
+sudo apt install libdbd-mysql-perl
+```
+
+To create the database and the database user (unless you keep an old database).
+Edit the command first if you want a non-default database name, user name or
+password.
+
+> **Note**: The Clickhouse MySQL interface requires a [double SHA1
+> password](https://clickhouse.com/docs/en/operations/settings/settings-users#password_double_sha1_hex)
+> ```
+> PASSWORD=$(base64 < /dev/urandom | head -c8); echo "$PASSWORD"; echo -n "$PASSWORD" | sha1sum | tr -d '-' | xxd -r -p | sha1sum | tr -d '-'

```

Comment:
Using `openssl` might be a bit less hacky. I also suggest avoiding `echo -n`:
```suggestion
> PASSWORD=$(openssl rand -base64 12); echo "$PASSWORD"; printf "%s" "$PASSWORD" | openssl sha1 -binary | openssl sha1 | awk '{ print $2 }'
```

Reply:
The oneliner is a copy/paste from the Clickhouse documentation.